apps/embed/templates/meinberlin_embed/embed.html

```diff
@@ -14,6 +14,7 @@
     <script src="{% static 'meinberlin.js' %}"></script>
     <script src="{% static 'embed.js' %}"></script>
     <meta name="viewport" content="width=device-width" />
+    <base target="_top" />
```

Comment:
I'm still not sure this is a good idea? Not only does it navigate away from the page containing the iframe (which I'd find annoying as a user) but navigating away from that page also resets the state of the iframe should one want to go back, removing any input (i.e. a comment) the user might've prepared but not sent yet.